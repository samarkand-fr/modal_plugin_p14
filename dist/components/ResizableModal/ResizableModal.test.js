"use strict";

var _react = _interopRequireDefault(require("react"));
var _react2 = require("@testing-library/react");
var _ResizableModal = _interopRequireDefault(require("./ResizableModal"));
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
var defaultProps = {
  onClose: jest.fn(),
  initialWidth: 400,
  initialHeight: 300,
  minWidth: 200,
  minHeight: 150,
  maxWidth: 800,
  maxHeight: 600
};
it('renders correctly', function () {
  (0, _react2.render)( /*#__PURE__*/_react.default.createElement(_ResizableModal.default, defaultProps, "Content"));
  var resizableModal = _react2.screen.getByTestId('resizable-modal');
  expect(resizableModal).toBeInTheDocument();
});