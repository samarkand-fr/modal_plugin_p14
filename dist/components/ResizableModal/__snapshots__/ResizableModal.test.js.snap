// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResizableModal renders correctly 1`] = `
<div>
  <div
    class="resizable-modal "
    style="width: 400px; height: 300px;"
  >
    <header
      class="modalHeader "
    >
      <button
        aria-label="Close"
        class="modalCloseButton "
        data-dismiss="modal"
        type="button"
      >
        <i
          class="fas fa-times"
        />
      </button>
    </header>
    <section
      class="modalSection "
    >
      Content
      <button
        class="close-button"
      >
        Close
      </button>
    </section>
    <footer
      class="modalFooter "
    />
    <div
      class="resizable-handle"
    />
  </div>
</div>
`;
